docs/sources/installation/kubernetes.md

```diff
@@ -0,0 +1,233 @@
+```bash
+kubectl create secret generic ge-license --from-file=/path/to/your/license.jwt
+```
+
+### Create Grafana Enterprise configuration
+Create a Grafana configuration file with the name `grafana.ini`. Then paste the content below. 
+>**Note:** You will have to update the `root_url` field to the url associated with the license you were given. 
+```yaml
+[enterprise]
+license_path = /etc/grafana/license/license.jwt
+[server]
+root_url =/your/license/root/url
+
+```
+
+### Create Configmap for Grafana Enterprise Config
+Create a Kubernetes Configmap from your `grafana.ini` file with the following command:
+```bash
+kubectl create configmap ge-config --from-file=/path/to/your/config.ini
+```
+### Create Grafana Enterprise Kubernetes manifest
+Create a `grafana.yaml` file, then paste the content below. This YAML is identical to the one for Grafana OS install except for the additional references to the Configmap which has your Grafana configuration file and the Secret that has your license. 
+
+```yaml
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: grafana
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: local-path
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: grafana
+  name: grafana
+spec:
+  selector:
+    matchLabels:
+      app: grafana
+  template:
+    metadata:
+      labels:
+        app: grafana
+    spec:
+      containers:
+        - image: grafana/grafana-enterprise:latest
+          imagePullPolicy: IfNotPresent
+          name: grafana
+          ports:
+            - containerPort: 3000
+              name: http-grafana
+              protocol: TCP
+          readinessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /robots.txt
+              port: 3000
+              scheme: HTTP
+            initialDelaySeconds: 10
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 2
+          resources:
+            limits:
+              memory: 4Gi
+            requests:
+              cpu: 100m
+              memory: 2Gi
+          volumeMounts:
+            - mountPath: /var/lib/grafana
+              name: grafana
+            - mountPath: /etc/grafana
+              name: ge-config
+            - mountPath: /etc/grafana/license
+              name: ge-license
+      volumes:
+        - name: grafana
+          persistentVolumeClaim:
+            claimName: grafana
+        - name: ge-config
+          configMap:
+            name: ge-config
+        - name: ge-license
+          secret:
+            secretName: ge-license
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
+spec:
+  ports:
+    - port: 3000
+      protocol: TCP
+      targetPort: http-grafana
+  selector:
+    app: grafana
+  sessionAffinity: None
+  type: LoadBalancer
+```
+ 
+1. Send manifest to Kubernetes API Server
```

Comment:
@achatterjee-grafana here I changed this to a 1. based on your suggestion, but above I have this as a heading with 3 #'s.